### PR TITLE
docs: broken link to carbon documentation

### PR DESCRIPTION
**Pull request by Kiittyka**
Closes #

This PR fixes a broken link to Carbon Docs site in README.md.

#### Changelog

**Changed**

- Changed the existing link (leads to a 404 error) to a working link.

#### Testing / Reviewing

It can be verified by navigating to the changed link.

### CODE REVIEW

**Comment by Kiittyka** [comment]
I have read the DCO document and I hereby sign the DCO.

**Comment by Kiittyka** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -61,8 +61,7 @@ we support!
 
 ## :books: Documentation
 
-- See our documentation site
-  [here](https://www.carbondesignsystem.com/get-started/develop/react) for full
+- See our documentation site [here](https://www.carbondesignsystem.com) for full
```
Comment (joshblack) [inline README.md]:
```suggestion
- See our documentation site [here](https://www.carbondesignsystem.com/developing/frameworks/react/) for full
```

I bet we can use this link to mirror the get started one that was originally there 👀

**Review by joshblack — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Kiittyka) [reply README.md]: Oh yeah, will change it accordingly. Thanks!

**Review by tay1orjones — APPROVED** [review]

**Comment by joshblack** [comment]
@Kiittyka it seems like the DCO bot is complaining about:

> Krithika S Udupa seems not to be a GitHub user. You need a GitHub account to be able to sign the DCO. If you have already a GitHub account, please add the email address used for this commit to your account.

Looking at the commits, they may have been used using an internal GitHub ID or the email address used is not associated with a GitHub account 👀 

image

**Comment by Kiittyka** [comment]
@joshblack I have added the GitHub Id used in the commits to my account now. But after a recheck, it still failed. So how do I go about it?

**Comment by joshblack** [comment]
@Kiittyka when looking at the commits in the commits tab, the profile doesn't seem to be showing up still. Tried to show what I mean in this video:


https://user-images.githubusercontent.com/3901764/129042752-4c25980d-5769-4533-bea9-389a2e9b907f.mov

**Comment by Kiittyka** [comment]
okay understood the issue. Is it okay to close this PR and create a new one?

**Comment by Kiittyka** [comment]
I have resolved the issue and have created a new PR for the same (#9467) . Thanks for the review @tay1orjones @joshblack !

**Pull request closed without merging**